tests/test_ecommerce.py

```diff
@@ -465,18 +465,30 @@ def test_metadata():
                     "enum": ["httpResponseBody", "browserHtml"],
                 },
                 "crawl_strategy": {
-                    "default": "full",
+                    "default": "default",
                     "description": "Determines how the start URL and follow-up URLs are crawled.",
                     "enumMeta": {
+                        "default": {
+                            "description": (
+                                "Follow pagination, subcategories, and product detail pages. "
+                                "If starting on a homepage, it would attempt to discover other "
+                                "URLs in the page using heuristics."
+                            ),
+                            "title": "Default",
+                        },
                         "full": {
-                            "description": "Follow most links within the domain of URL in an attempt to discover and extract as many products as possible.",
+                            "description": (
+                                "(Deprecated. Use Default instead) Follow most links "
```

Comment:
I'm not even sure we should be deprecating those 2 extra strategies :) What do you think about not getting them deprecated now, and getting a bit more feedback?

Reply:
Ahhh this is what @Gallaecio has recently raised and discussed in the call today. 👍 Yeah, we could remove marking them as deprecated and still expose them to users. 

Adrian's main concern was about making `navigation` still available to users in case a given website works quite well with it _(without heuristics)_. If so, they could reduce the number of requests due to having less requests due to heuristics.

Reply:
@BurnzZ suggested an alternative approach, which I think is worth considering: have a separate boolean/enum field for heuristics.

So, we could make the crawl strategy be about the crawl strategy only, not heuristics. We could deprecate `full` (because in terms of crawl strategy, `full` is equivalent to `navigation` pointed to a homepage), and keep `navigation`, `pagination-only`, and a future `direct-item`. Then have a separate Heuristics field with options like `follow-all-homepage` (this new approach), `follow-all-navigation` (equivalent to the `full` old crawl strategy, should be default for backward compatibility at least in pre-existing virtual spiders), `none` (equivalent to the old `navigation`), `sitemap`, etc.

I think it would provide a better user experience overall, but it might make backward-compatibility a bit harder to accomplish.

Reply:
Do we need to handle it in this PR? On a first sight, we can just add a new strategy for now, make it default, and then we can think about all the deprecations, backwards compatibility, new options, etc.

Reply:
Would this approach make the transition any worse?

Reply:
I think and extra field would indeed negatively affect the transition. With the strategy field only, existing virtual spiders in Scrapy Cloud can maintain backward compatibility as long as Scrapy Cloud stored the actual value selected (even if it was the default), which seems to be the case.

With a new field that Scrapy Cloud knows nothing about, whose default value would cause a backward-incompatible behavior, keeping the old behavior in pre-existing virtual spiders in Scrapy Cloud would be much harder, I think.